pkg/controller/util/node/controller_utils.go

```diff
@@ -121,7 +119,7 @@ func SetPodTerminationReason(kubeClient clientset.Interface, pod *v1.Pod, nodeNa
 func MarkPodsNotReady(kubeClient clientset.Interface, recorder record.EventRecorder, pods []*v1.Pod, nodeName string) error {
 	klog.V(2).InfoS("Update ready status of pods on node", "node", nodeName)
 
-	errMsg := []string{}
```

Comment:
s/errrors/errs/g

To avoid conflict with `github.com/pkg/errors` packages in the future.

Reply:
Agree.